src/test/java/de/andrena/tools/nopackagecycles/NoPackageCyclesRuleIntegrationTest.java

```diff
@@ -24,38 +24,45 @@
 	private NoPackageCyclesRule rule;
 	private EnforcerRuleHelperMock helper;
 
+
 	@Before
 	public void setUp() throws Exception {
 		rule = new NoPackageCyclesRule();
 		helper = new EnforcerRuleHelperMock();
 	}
 
+
 	@Test
 	public void fitnesseIntegrationTest() throws Exception {
 		assertPackageCycles(FITNESSE_TARGET_FOLDER, FITNESSE_EXPECTED_OUTPUT);
 	}
 
+
 	@Test
 	public void junitIntegrationTest() throws Exception {
 		assertPackageCycles(JUNIT_TARGET_FOLDER, JUNIT_EXPECTED_OUTPUT);
 	}
 
+
 	private void assertPackageCycles(URL targetFolder, URL expectedOutput) throws URISyntaxException, IOException {
 		helper.setTestClassesDir(new File("non-existent"));
 		helper.setClassesDir(new File(targetFolder.toURI()));
 		try {
 			rule.execute(helper);
 			fail("expected EnforcerRuleException");
-		} catch (EnforcerRuleException e) {
+		}
+		catch (EnforcerRuleException e) {
 			// using assertEquals to get a nice comparison editor in eclipse
-			assertEquals(getExpectedOutput(expectedOutput), e.getMessage());
+			assertEquals(getExpectedOutput(expectedOutput), e.getMessage() + "\n");

```

Comment:
Tests will fail in IntelliJ if not. Will work via mvn build with or without linebreak.